backend/views/syson-diagram-common-view/src/main/java/org/eclipse/syson/diagram/common/view/nodes/AbstractUsageNodeDescriptionProvider.java

```diff
@@ -109,7 +109,7 @@ public AbstractUsageNodeDescriptionProvider setPrecondition(String aPrecondition
      *         the cache used to retrieve node descriptions.
      * @return the set of {@link NodeDescription} that are added as reused child.
      */
-    protected abstract Set<NodeDescription> getBorderNodes(IViewDiagramElementFinder cache);
+    protected abstract Set<NodeDescription> getReusedBorderNodes(IViewDiagramElementFinder cache);
```

Comment:
that is reused as border node in that Usage node description. => that are reused as border nodes

 that are added as reused child. => that are reused as border nodes.